### ISSUE TITLE
Multiple modals are created if you click the photo behind an open modal

### ISSUE THREAD

**Issue by bbstilson** [luispcosta/Reddit-Image-Browser#1]
To repro: Select an image, click on other images behind the modal. This opens a new modal rather than replacing the current open one. You can verify this by closing the current open modal.

The user shouldn't be able to select another photo without closing the current expanded photo.

**Comment by luispcosta** [comment]
Yes! Yes they are. Sorry for not responding early enough. This ticket is on my todo list. I would be happy if you submitted a PR though!
